docs/Contributing.md

```diff
@@ -151,6 +151,19 @@ We do not want to rely on the Vulkan CTS. Running the Vulkan CTS takes a long
 time, putting a *time-to-test* burden on all developers. The Vulkan CTS tests
 are also harder to debug because more components are involved and require a GPU.
 
+LIT tests (tests that are run using the `llvm-lit` tool) should provide good
+and reliable coverage, e.g. using `ĈHECK-NEXT` lines, while still being easy
```

Comment:
The C of `CHECK-NEXT` has a circumflex on it?

Reply:
Dead keys... fixed.